end-user-surveys/analysis-guides/survey-analysis.md

```diff
@@ -0,0 +1,189 @@
+
+**Cleaning 1: Frequency & Percentage Check**
+
+Before editing anything, use Pivot Tables to generate frequency and percentage tables for each variable.
+This helps you quickly spot:
+
+ - Unfinished responses and missing values (blank cells)
+ - Inconsistent categories (e.g., “AWS Fargate” vs “aws fargate”)
+ - Outliers or unexpected entries
+
+***Why this matters: This simple step often surfaces most data issues early and informs how you’ll clean each variable***.
+
+Note: 
+
+ - Using pivot tables to get frequencies and percentages should not be used for multiple response variables and open-ended variables.
+ - Some blank cells can be due to the conditional formatting of the form.
+
+**Steps to using pivot tables:** 
+
+
+
+https://github.com/user-attachments/assets/ce119f86-07db-4946-8efc-d6c78a293548
+
+
+https://github.com/user-attachments/assets/774df128-c6b9-46f1-b763-8d04b40fb4c4
+
+
+
+**Cleaning 2: Standardize Responses**
+
+ - Category Names: Use consistent capitalization and naming (e.g., “AWS Fargate” everywhere).
+ - Remove Duplicates: Check for duplicate rows, especially when respondents accidentally submit multiple times.
+
+## Data Transformation
+Once the data is clean, structure it for analysis. This involves creating unique identifiers and transforming variables for easier manipulation.
+
+**Step 1: Create a Respondent ID**
+
+Insert a new column named **respondent_id** and number each row sequentially from 1 to n.
+This ID will become the key for linking transformed data back to the original sheet.
+
+**Step 2: Single-Select Variables**
+
+Keep single-select categorical variables in their raw form initially.
+You’ll transform them later, only when needed for specific statistical tests or advanced analysis.
+
+**Step 3: Multiple-Select Variables**
+For questions where respondents can select multiple answers:
+ - ≤ 10 options:
+    - Convert each option into a dummy variable (0/1) using IF(ISERR(SEARCH(I$1,$H2)),0,1).
+![69D61F99-06A9-4EB5-B7B0-76EF895EC32D_1_105_c](https://github.com/user-attachments/assets/653bbb97-8d39-4c09-a880-4a8633cff97c)  
+    
+    - Each option becomes its own column (e.g., deployment_k8s, deployment_vm, etc.).
+      
+ - Greater than 10 options:
+    - To keep your sheet manageable, create a separate sheet for that question.
+    - Use the respondent_id to link it back when needed (e.g., through VLOOKUP)
+    - And this time, the only change in the formula is checking the matching case in the main data sheet, i.e, $O2 as shown in the screenshot, will be found in the data!$O2
+
+**Aggregation Tip:**
+
+ - For multi-select questions: use SUM() across the dummy columns to get counts per option.
+ - For single-select questions: in one of your Pivot Table sheets, get the counts and percentages.
```

Comment:
I like that you mention a specific function (SUM) in the point above. Perhaps you could reference a specific function here as well.